lumen/transforms/sql.py

```diff
@@ -10,6 +10,12 @@ class SQLTransform(Transform):
         
     __abstract = True
     
+    def __str__(self):
+        vals = self.param.get_param_values()
+        vals_dict = {k: v for k, v in vals}
+        vals_dict.pop('name', None)
```

Comment:
Does this pip the name from the original param values?

Reply:
`name` is a field inherited all the way from `param.Parametrized`. It usually looks something like MyTransformType10248 (w/ a random and unique id at the end). This interrupts the query based caching b/c this changing random number alters the hash even if no input parameters change. This `str` pops that `name` off and uses the result of the values for the hash.